src/blocks/author-list/components/inspector/StylePanel.js

```
@@ -0,0 +1,272 @@
+		<>
+			<PresetStyles
+				onApplyPreset={ ( settings ) => {
+					// Apply all preset settings at once
+					setAttributes( settings );
+				} }
+			/>
+
+			<PanelBody title={ __( 'List Background', 'author-profile-blocks' ) }>
+				<BaseControl
+					label={ __( 'Block Background', 'author-profile-blocks' ) }
+				>
```

ESLint

[error] When using BaseControl component if a label property is passed an id property should also be passed.

(@wordpress/no-base-control-with-label-without-id)